src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -6,7 +6,6 @@ export const frontendApps = {
   RX: 'rx',
   MESSAGING: 'messaging',
   LAB_AND_TEST_RESULTS: 'lab-and-test-results',
-  APPOINTMENTS: 'appointments',
```

Comment:
I wonder if it's still accurate to refer to these as `frontendApps`. Would `widgetTypes` be more appropriate with how they're intended to be used?

Reply:
good question. I'll make sure they are only being used as `widgetTypes` and make that change if they are.